cmd/kusk/cmd/root.go

```diff
@@ -147,3 +191,68 @@ func initConfig() {
 		fmt.Fprintln(os.Stderr, fmt.Errorf("failed to read config file %q, %w", viper.ConfigFileUsed(), err))
 	}
 }
+
+func init() {
+	cobra.OnInitialize(initConfig)
+
+	// Here you will define your flags and configuration settings.
+	// Cobra supports persistent flags, which, if defined here,
+	// will be global for your application.
+
+	// Cobra also supports local flags, which will only run
+	// when this action is called directly.
+	rootCmd.SetHelpFunc(help)
+}
+
+func help(c *cobra.Command, s []string) {
+
+	switch c.Use {
+	case "mock":
```

Comment:
Cannot you not use the `.Use` field in the different commands instead of `string`s in the `case` statements? E.g.,

`case mockCmd.Use`

Would work here I believe?

If it doesn't work, I'd just leave it as-is.